crates/vm/levm/src/opcode_handlers/arithmetic.rs

```diff
@@ -260,6 +260,27 @@ impl<'a> VM<'a> {
             Ok(OpcodeResult::Continue { pc_increment: 1 })
         }
     }
+
+    pub fn op_clz(&mut self) -> Result<OpcodeResult, VMError> {
+        if self.env.config.fork < Fork::Osaka {
+            return Err(ExceptionalHalt::InvalidOpcode.into());
+        }
+
+        let current_call_frame = &mut self.current_call_frame;
```

Comment:
you don't need a temporary anymore, some code has it due to the previous way of accessing the current callframe, accessing it directly is better as you dont create mutable referenceses